src/npc_class.cpp

```diff
@@ -244,6 +244,26 @@ void npc_class::load( JsonObject &jo, const std::string & )
         }
     }
 
+    /* Mutation rounds can be specified as either a single distribution:
+     *   "mutation_rounds" : { "constant" : 1 }
+     * or a map of categories to distributions:
+     *   "mutation_rounds" : [
```

Comment:
This example doesn't match the code, it looks more like:
```
"mutagen_rounds": [
  [ "MUTCAT_ANY", { "constant": 1 } ],
  [ "MUTCAT_INSECT", { "rng": [1, 3] } 
]
```
I prefer the syntax in the comment, is there a reason you changed it?